tests/ludwig/data/dataframe/test_dask.py

```diff
@@ -5,6 +5,7 @@
 from tests.integration_tests.utils import generate_data_as_dataframe
 
 
+@pytest.mark.slow
```

Comment:
I wonder what makes this test particularly slow, perhaps we can modify to use 1 feature and just train for 5 steps instead of an entire epoch